libsolidity/formal/CHC.cpp

```diff
@@ -521,6 +520,18 @@ void CHC::setCurrentBlock(smt::SymbolicFunctionVariable const& _block)
 	m_currentBlock = predicate(_block);
 }
 
+void CHC::setCurrentBlock(
```

Comment:
The annoying this is that this function is mostly the same as the one above, just changing the last line. Not sure it's worth it to pull that part out.